javascript-view-swapping/main.js

```diff
@@ -0,0 +1,23 @@
+const $tabContainer = document.querySelector('.tab-container');
+const $tabElements = document.querySelectorAll('.tab');
+const $views = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', function (event) {
+  if (event.target.matches('.tab')) {
```

Comment:
I recommend bailing out if `event.target` does _not_ match `.tab`. Otherwise, clicking to the right of the tabs will hide everything.